tests/python/pants_test/backend/python/tasks/test_resolve_requirements.py

```diff
@@ -40,7 +41,8 @@ def test_resolve_simple_requirements(self):
       self.assertIn("ModuleNotFoundError: No module named 'colors'", stderr_data)
     except AssertionError:
       # < Python 3.6 uses ImportError instead of ModuleNotFoundError.
-      self.assertIn("ImportError: No module named 'colors'", stderr_data)
+      # Python < 3 uses not quotes for module, python >= 3 does.
+      self.assertNotEqual(re.search(r"ImportError: No module named '?colors'?", stderr_data), None)
```

Comment:
The usual approach we would use is 

```python
from futures.utils import PY3

if PY2:
 self.assertIn("ImportError: No module named colors", stderr_data)
else:
  self.assertIn("ImportError: No module named 'colors'", stderr_data)
```

The reason is that it will allow us to more easily remove the Py2 code once we migrate by grepping `PY2` and `PY3`

Reply:
Fair point. I was trying not to add too much logic to the `except` block. 

I do think the test is very rigid, and should not fail for something as slight as missing quotes. For that reason (and the above), I feel a little more comfortable leaving the regexp in there.